Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -100,7 +100,9 @@ let Keyboard = {
    * - `keyboardDidChangeFrame`
    *
    * Note that if you set `android:windowSoftInputMode` to `adjustResize`  or `adjustNothing`,
-   * only `keyboardDidShow` and `keyboardDidHide` events will available on Android.
+   * only `keyboardDidShow` and `keyboardDidHide` events will be available on Android and
+   * `keyboardWillShow` as well as `keyboardWillHide` are generally not available on Android
+   * since there is no corresponding event.
```

Comment:
The relationship between `android:windowSoftInputMode`, [`keyboardDidShow` + `keyboardDidHide`], and  [`keyboardWillShow` + `keyboardWillHide`] is still unclear to me. Does  `android:windowSoftInputMode` affect  [`keyboardWillShow` + `keyboardWillHide`], or this pair of events generally unavailable on Android?